memory_addr/src/range.rs

```diff
@@ -46,13 +47,76 @@ where
     /// assert_eq!(range.start, 0x1000);
     /// assert_eq!(range.end, 0x2000);
     /// ```
+    ///
+    /// And this will panic:
+    ///
+    /// ```should_panic
+    /// # use memory_addr::AddrRange;
+    /// let _ = AddrRange::new(0x2000usize, 0x1000);
+    /// ```
+    #[inline]
+    pub fn new(start: A, end: A) -> Self {
```

Comment:
`new` and `try_new` can be const?

Reply:
They cannot because `start <= end` is not.